kubernetes/windows/Dockerfile

```diff
@@ -94,4 +93,17 @@ ENV AZMON_COLLECT_ENV False
 ENV CI_CERT_LOCATION "C://oms.crt"
 ENV CI_KEY_LOCATION "C://oms.key"
 
+# Delete unnecessary files after installation
+RUN powershell -Command "Remove-Item -Recurse -Force 'C:\ruby31\share\doc\ruby\html\js'; \
+Remove-Item -Force 'C:\ruby31\ridk_use\ridk.ps1'; \
+Remove-Item -Force 'C:\ruby31\msys64\usr\share\texinfo\js\info.js'; \
+Remove-Item -Force 'C:\ruby31\msys64\usr\share\texinfo\js\modernizr.js'; \

```

Comment:
either we should clean up entire \usr\share\textinfo directory or use wild card 

Remove-Item -Force 'C:\ruby31\msys64\usr\share\texinfo\js\*.js'

Reply:
changed